tests/tests_core/test_early_stopping.py

```diff
@@ -57,16 +57,16 @@ def test_group_sequential(self):
         """
         Check the group sequential function.
         """
-        stop, delta, CI, n_x, n_y, mu_x, mu_y = es.group_sequential(self.rand_s1, self.rand_s2)
-        self.assertEqual(stop, 1)
-        self.assertAlmostEqual(delta, -0.15887364780635896)
-        # np.testing.assert_almost_equal(CI.values(), [-0.24461812530841959, -0.07312917030429833], decimal=5)
-        np.testing.assert_almost_equal(CI[97.5], -0.07312917030429833, decimal=5)
-        np.testing.assert_almost_equal(CI[2.5000000000000022], -0.24461812530841959, decimal=5)
-        self.assertEqual(n_x, 1000)
-        self.assertEqual(n_y, 1000)
-        self.assertAlmostEqual(mu_x, -0.045256707490195384)
-        self.assertAlmostEqual(mu_y, 0.11361694031616358)
+        res = es.group_sequential(self.rand_s1, self.rand_s2)
+
+        self.assertEqual               (res['stop'],            True)
+        self.assertAlmostEqual         (res['delta'],          -0.15887364780635896)
+        np.testing.assert_almost_equal (res['interval'][02.5], -0.24461812530841959, decimal=5)
+        np.testing.assert_almost_equal (res['interval'][97.5], -0.07312917030429833, decimal=5)
+        self.assertEqual               (res['n_x'],             1000)
+        self.assertEqual               (res['n_y'],             1000)
+        self.assertAlmostEqual         (res['mu_x'],           -0.045256707490195384)
+        self.assertAlmostEqual         (res['mu_y'],            0.11361694031616358)
```

Comment:
I agree that writing this might be a bit cumbersome, but reading is much easier - and we tend to read more often than write.